__tests__/app.test.js

```diff
@@ -18,6 +18,18 @@ afterAll(() => {
   db.end();
 });
 
+describe.only("GET /api", () => {
```

Comment:
Please be very careful not to leave .onlys on your tests when committing. Husky is in place to ensure that all of your tests pass while committing so we aren't pushing broken code.